packages/frontend/src/components/HeaderMenu/HeaderMenu.tsx

```diff
@@ -10,19 +11,29 @@ const HeaderMenu: FunctionComponent = () => {
   const { disconnect } = useDisconnect();
   const { icon, text } = useWalletBranding();
 
+  const navigate = useNavigate();
+
   if (!address || !isConnected) return null;
 
   const label = ensName ? formatEnsName(ensName) : formatAddress(address);
 
   const menuLinks = [
+    {
+      title: 'Dashboard',
+      onClick: () => navigate('/dashboard'),
```

Comment:
Let's change this to an `href` prop. The Menu component accepts this as well.

Reply:
3d8dd91